.deploystack/deploystack.json

```diff
@@ -0,0 +1,15 @@
+{	"title":"Microservices Demo (Online Boutique)",
+	"duration":5,
+	"documentation_link": "https://cloud.google.com/shell/docs/cloud-shell-tutorials/deploystack/microservices-demo",
+	"collect_project":true,
+	"collect_region":true,
+    "collect_billing_account":false,
+	"region_type":"compute",
+	"region_default":"us-central1",
+	"hard_settings":{
+		"memorystore":"false",
+        "name": "online-boutique",
+        "namespace" : "default", 
+        "filepath_manifest" : "../release/kubernetes-manifests.yaml"
+	}
+}
```

Comment:
**Question:**
Do you mind if I go ahead and convert this to YAML?

Reply:
Did it for you.